scripts/dev/create_package.sh

```diff
@@ -284,10 +284,10 @@ function createPackage {
         rm -Rf utils/fileBufferLoadingCSVExample
         rm -Rf 3d/modelNoiseExample
         rm -Rf windowing
-		rm -Rf input_output
-		rm -Rf shader
-		rm -Rf sound
-		rm -Rf threads
+	rm -Rf input_output
```

Comment:
it's tabs right - 2 spaces width, I think the clangformat says otherwise could be causing issues with tab /space style